templates/committee/_layout.html.twig

```diff
@@ -16,10 +16,12 @@
                     <div class="text--uppercase text--small b__nudge--bottom-tiny">
                         Comité local {% if is_host %}n°{{ committee.id }}{% endif %}
                     </div>
-                    <h2 class="text--large">
+                    <div class="committee__name">
                         {{ include('components/location-pin.html.twig') }}
-                        <span id="committee-name" class="committee__name">{{ committee.name }}</span>
-                    </h2>
+                        <h1 id="committee-name" class="text--large">
```

Comment:
can you try apply this class to the h1 `b__nudge--left-small` [source](https://github.com/EnMarche/en-marche.fr/blob/580fdd180df12c669917902a823610d8021c5c32/front/style/layout/_layout.scss#L415)